CallStatusNotifier/WindowsTray.cs

```diff
@@ -0,0 +1,219 @@
+                seperatorMain,
+                triggerBuzzer,
+                seperator,
+                exit
+            });
+
+            startNotifier.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            startNotifier.Text = "Start Notifier";
+            startNotifier.Click += new EventHandler(StartNotifierClicked);
+
+            endNotifier.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            endNotifier.Text = "End Notifier";
+            endNotifier.Click += new EventHandler(EndNotifierClicked);
+
+            triggerBuzzer.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            triggerBuzzer.Text = "Trigger Buzzer";
+            triggerBuzzer.Click += triggerBuzzer_Click;
+
+            exit.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            exit.Text = "Exit";
+            exit.Click += new EventHandler(ExitClicked);
+        }
+
+        void triggerBuzzer_Click(object sender, EventArgs e)
+        {
+            TriggerBuzzer();
+        }
+
+        void ExitClicked(object sender, EventArgs e)
+        {
+            System.Windows.Forms.Application.Exit();
+        }
+
+        void EndNotifierClicked(object sender, EventArgs e)
+        {
+            ChackHostNamesAndEndNotifier();
+        }
+
+        void StartNotifierClicked(object sender, EventArgs e)
+        {
+            StartPhotonAndNotify();
+        }
+
+        public async Task StartPhotonAndNotify()
+        {
+            await LoginToCloudAndGetDevice();
+            if (deviceSuccessfullyInitiated)
+            {
+                string myFunction = myDevice.Functions[0];
+                ParticleFunctionResponse functionResponse = await myDevice.RunFunctionAsync(myFunction, string.Format("on,{0}", Environment.UserName));
+            }
+        }
+
+        public async Task EndNotifier(bool clearHosts = false)
+        {
+            string myFunction = myDevice.Functions[0];
+            ParticleFunctionResponse functionResponse = await myDevice.RunFunctionAsync(myFunction, string.Format("off,{0}", Environment.UserName));
+            if (clearHosts)
+            {
+
+                string clearHostFunction = myDevice.Functions[1];
+                await myDevice.RunFunctionAsync(clearHostFunction, string.Empty);
+            }
+            ParticleCloud.SharedCloud.Logout();
+        }
+
+        private async Task LoginToCloudAndGetDevice()
+        {
+            ParticleCloud.SharedCloud.SynchronizationContext = System.Threading.SynchronizationContext.Current;
+            var success = await ParticleCloud.SharedCloud.TokenLoginAsync("c078178825f98c72c7ea62fd3eb0cc81207553db");
```

Comment:
Consider adding this token into a resource file
ref: https://msdn.microsoft.com/en-us/library/aa984334(v=vs.71).aspx